app/student/forms.py

```diff
@@ -1,77 +1,167 @@
+from flask import url_for
+import calendar
+from datetime import datetime
 from flask_wtf import Form
 from wtforms import ValidationError
 from wtforms.ext.sqlalchemy.fields import QuerySelectField
 from wtforms.fields import (PasswordField, StringField, SubmitField,
-                            IntegerField, FloatField, SelectField, BooleanField)
-from wtforms.fields.html5 import EmailField
+							IntegerField, FloatField, SelectField, BooleanField)
+from wtforms.fields.html5 import EmailField, DateField
 from wtforms.validators import Email, EqualTo, InputRequired, Length
+from wtforms.fields.html5 import EmailField
+
+class EditCommonAppEssayForm(Form):
+	link = StringField(
+		'Link to Common App Essay', validators=[InputRequired(), Length(1, 64)])
+	status = SelectField(
+		'Status', choices=[('Incomplete','Incomplete'), 
+							('Waiting', 'Waiting'), 
+							('Reviewed', 'Reviewed'),
+							('Edited', 'Edited'),
+							('Done', 'Done')], 
+							validators=[InputRequired()])
+	submit = SubmitField('Update essay link')
+
+class EditCollegeForm(Form):
+	college_name = StringField(
+		'College Name', validators=[InputRequired(), Length(1, 64)])
+	submit = SubmitField('Update college name')
+
+class EditTestScoreForm(Form):
+	month_choices = []
+	for i in range(1, 13):
+		month_choices.append((calendar.month_name[i], calendar.month_name[i]))
+	year_choices = []
+	today = datetime.today()
+	for i in range(0, 8):
+		year_choices.append((str(today.year - 8 + i), str(today.year - 8 + i)))
+	test_name = StringField(
+		'Test Name', validators=[InputRequired()])
+	month = SelectField(u'Month', choices=month_choices, validators=[InputRequired()])
+	year = SelectField(u'Year', choices=year_choices, validators=[InputRequired()])
+	score = IntegerField('Test Score', validators=[InputRequired()])
+	submit = SubmitField('Update test score')
+
+class EditSupplementalEssayForm(Form):
+	essay_name = StringField(
+		'Essay Name', validators=[InputRequired(), Length(1, 64)])
+	link = StringField(
+		'Essay Link', validators=[InputRequired(), Length(1, 64)])
+	status = SelectField(
+		'Status', choices=[('Incomplete','Incomplete'), 
+							('Waiting', 'Waiting'), 
+							('Reviewed', 'Reviewed'),
+							('Edited', 'Edited'),
+							('Done', 'Done')], 
+							validators=[InputRequired()])
+	submit = SubmitField('Update essay')
+
+class AddChecklistItemForm(Form):
+	item_text = StringField(
+		'Checklist Item', validators=[InputRequired(), Length(1, 64)])
+	date = DateField('Deadline', format='%Y-%m-%d', validators=[InputRequired()])
```

Comment:
Is %Y-%m-%d the way we're going to be rendering deadlines? Or is this just arbitrary to be parsed later on?

Reply:
I just included that because I thought it was required for rendering the DateField, but when I took it out it seemed to behave the same way!